### DISCUSSION TITLE
Separate memory slots for each band.

### DISCUSSION THREAD

**Discussion by Overland-DX** [PE5PVB/TEF6686_ESP32#151] [Ideas]
First of all, thank you so much for the amazing job you all are doing on the firmware for this tuner.
And I am very happy that we have received 30 memory slots.
But is it possible to separate them for each band?
Such as for example:
LW: 5 slots,
MW: 10 slots,
SW: 30 slots,
Oirt: 5 to 10 slots,
FM: 10 to 20 slots

I know you guys are already pushing the esp32 table a lot, but it's just a suggestion.

And another thing,
Is it possible to show in a way whether the memory slot is in use or the memory slot is free, when you save a new one? Because sometimes I have a little doubt whether it is slot 12 or 13 that I saved a frequency on last.

**Comment by andimik** [comment]
You already can use all 30 mem places for SW!

**Comment by Overland-DX** [comment]
I haven't said anything else!
But it is easier to organise, when they are split up and if there is room for it, I think many people would appreciate more memory slots.
Last but not least, it's just a suggestion referencing as the topic "ideas"

**Comment by PE5PVB** [comment]
The idea is to have mixed memory channels, so you don't have to switch the band first before you can call the memory channel of your choice.

**Reply by Overland-DX** [reply]
Thank you for the answer. If it's meant to be that way, I'm totally happy with it.
I'm just used to it from other radios that they split.
